data/tips.cfg

```diff
@@ -1,15 +1,15 @@
 #textdomain wesnoth
 
 [tip]
-    text= _ "Use the map editor's <i>Areas</i> menu to define regions later addressed by event or AI coding."
+    text= _ "Use the map editor’s <i>Areas</i> menu to define regions later addressed by event or AI coding."
     source= _ "<i>― The Wesnoth Design Guide</i>"
 [/tip]
 [tip]
-    text= _ "Create a <i>New Scenario</i> in the editor's <i>File</i> menu to make use of advanced features."
+    text= _ "In the map editor, create a <i>New Scenario</i> in the <i>File</i> menu to make use of advanced features."
```

Comment:
The change is good, and I suggest a further change, s/create a/select/, because I think the instruction would be clearer this way.